patches/net/minecraft/server/level/ChunkMap.java.patch

```diff
@@ -1,5 +1,14 @@
 --- a/net/minecraft/server/level/ChunkMap.java
 +++ b/net/minecraft/server/level/ChunkMap.java
+@@ -135,7 +_,7 @@
+     private final String storageName;
+     private final PlayerMap playerMap = new PlayerMap();
+     private final Int2ObjectMap<ChunkMap.TrackedEntity> entityMap = new Int2ObjectOpenHashMap<>();
+-    private final Long2ByteMap chunkTypeCache = new Long2ByteOpenHashMap();
++    private final Long2ByteMap chunkTypeCache = new net.neoforged.neoforge.common.util.SizeRestrictedLong2ByteLinkedOpenHashMap(net.neoforged.neoforge.common.NeoForgeConfig.SERVER.chunkTypeCacheLimit.get()); // Neo: PR #937
```

Comment:
Why not just remove entries from this map in the same spot you patch in `PointOfInterestUnloading.onChunkUnload`? I have concerns about the performance of repopulating this cache at runtime if needed.